containerd/main.go

```diff
@@ -137,6 +142,13 @@ func main() {
 				}
 			}
 		}
+		if logLevel := context.GlobalString("log-level"); logLevel != "" {
+			lvl, err := logrus.ParseLevel(logLevel)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "Unable to parse logging level: %s\n", logLevel)
+			}
+			logrus.SetLevel(lvl)
```

Comment:
On error `lvl` will be set to panic which is a bit useless, please set it to `info` in the if above